src/main/java/io/anserini/search/SearchTweets.java

```diff
@@ -58,7 +58,7 @@
   private SearchTweets() {}
 
   public static void main(String[] args) throws Exception {
-    long curTime = System.nanoTime();
+    long startQueryTime = System.currentTimeMillis();
```

Comment:
In light of a later comment, I missed that there were two timers. This timer isn't really measuring query time, so is it far to call it `startQueryTime`?